scRNA-seq/04-tag-based_scRNA-seq_processing.Rmd

```diff
@@ -0,0 +1,246 @@
+However, Drop-seq data is also supported, and in this case you would use a `--dropseq` flag instead of this.
+
+#### `--tgMap`
+This is needed to supply a transcript to gene key that Alevin will use to quantify the genes.
+For our example, we've pre-made the file `Mus_musculus.GRCm38_tx2gene.tsv` from the Ensembl transcriptome that we indexed above. 
+The file has to be a tsv file.
+
+
+#### `--dumpFeatures`
+This option will print out information that we will need for quality checks
+later on, including files with information on the UMIs and cell barcodes.
+
+See the [Alevin documentation](https://salmon.readthedocs.io/en/latest/alevin.html) for a complete list of the Alevin options.
+There are also a number of example analyses at the [Alevin tutorial](https://combine-lab.github.io/alevin-tutorial/#blog) website.
+
+
+### Note: Running the FULL sample.
+
+Sequencing runs are often split into multiple `fastq` files, both when a sample was run across multiple lanes and to keep the individual file sizes down. 
+This was the case for the *Tabula Muris* data we used here as well. If you look in the `data/tabula-muris/fastq-raw/10X_P4_3` directory, you will see the following files, representing 2 lanes, with three R1 and three R2 files per lane:
+
+```
+10X_P4_3_L001_R1_001.fastq.gz  10X_P4_3_L002_R1_001.fastq.gz
+10X_P4_3_L001_R1_002.fastq.gz  10X_P4_3_L002_R1_002.fastq.gz
+10X_P4_3_L001_R1_003.fastq.gz  10X_P4_3_L002_R1_003.fastq.gz
+10X_P4_3_L001_R2_001.fastq.gz  10X_P4_3_L002_R2_001.fastq.gz
+10X_P4_3_L001_R2_002.fastq.gz  10X_P4_3_L002_R2_002.fastq.gz
+10X_P4_3_L001_R2_003.fastq.gz  10X_P4_3_L002_R2_003.fastq.gz
+```
+
+We should really run all of these through Salmon, though that will take ~six times as long as the single pair of reads we used. 
+To do this, we could list each R1 and R2 file (space separated) after the `-1` and `-2` arguments, respectively.
+But that is a lot of typing, so a nice shortcut is to use a `*` character to represent a wildcard that will be filled in with whatever characters are present in the files at the given path.
+In the pattern `10X_P4_3_L*_R1_*.fastq.gz`, this would allow any lane number and any subset, so would match all of the following files (all the R1 files in this case):
+
+```
+10X_P4_3_L001_R1_001.fastq.gz  10X_P4_3_L002_R1_001.fastq.gz
+10X_P4_3_L001_R1_002.fastq.gz  10X_P4_3_L002_R1_002.fastq.gz
+10X_P4_3_L001_R1_003.fastq.gz  10X_P4_3_L002_R1_003.fastq.gz
+```
+
+For this directory, that would look make our full `salmon alevin` command look like this (don't run this now!):
+
+```
+salmon alevin -l ISR \
+  -i /shared/data/reference/refgenie/mm10_cdna/salmon_index/short/short/ \
+  -1 data/tabula-muris/fastq-raw/10X_P4_3/10X_P4_3_L*_R1_*.fastq.gz \
+  -2 data/tabula-muris/fastq-raw/10X_P4_3/10X_P4_3_L*_R2_*.fastq.gz \
+  --chromium  \
+  -p 4 \
+  -o data/tabula-muris/alevin-quant/10X_P4_3_full \
+  --tgMap data/tabula-muris/mm_ensdb95_tx2gene.tsv \
+  --dumpFeatures
```

Comment:
Sometimes we'll take an additional step of commenting out things we don't people to run such that even if they copy and pasted the code block it would not run.

Reply:
Ah, like we did with the index step above.

Reply:
I just wrote out this note before seeing Jackie already said it. I occur. Let's put the safety on.

Reply:
Will do.